perf/benchmark/runner/runner.py

```diff
@@ -108,7 +110,6 @@ def __init__(
         self.run_clientsidecar = clientsidecar
         self.run_bothsidecar = bothsidecar
         self.run_ingress = ingress
-        self.cacert = cacert
```

Comment:
No cacert?

Reply:
No; cacert moved to `test-server.crt` in `fortio.yaml` below. Does that suffice?